src/renderer/rendererutils.ts

```diff
@@ -26,24 +7,16 @@
 
     // Not sure how we can decide who won or lost yet. 
     // Combat log doesn't make it obvious.
-    if ((category == "Battlegrounds") || (category == "Solo Shuffle")) {
-        return "";
-    }
+    switch (category) {
+        case VideoCategory.MythicPlus:
+            return isGoodResult ? "Timed" : "Fail";
```

Comment:
Deplete better than fail?

Reply:
Yes, that's what I was looking for! Thanks.

Reply:
Fixed in 8a2d3cf